### ISSUE TITLE
[Audit] Enable audit logging in local keycloak containers

### ISSUE THREAD

**Issue by JonasCir** [hzi-braunschweig/SORMAS-Docker#371]
Hi there,
with https://github.com/hzi-braunschweig/SORMAS-Project/pull/8900 being merged, please follow the instructions provided [here](https://github.com/hzi-braunschweig/SORMAS-Project/blob/development/sormas-base/doc/keycloak.md#audit-logging) i.e.,

> To enable audit loging, please copy the files from `sormas-base/setup/keycloak/audit-logging/ to /opt/jboss/startup-scripts/` to `/opt/jboss/startup-scripts/` inside your Keycloak server and restart the server.

I'm not certain on which Keycloak server config file you rely (`standalone.xml` or `standalone-ha.xml`) so I provided logger files for both [here](https://github.com/hzi-braunschweig/SORMAS-Project/tree/development/sormas-base/setup/keycloak/audit-logging) .

**Comment by fhauptmann** [comment]
Please make sure the needed files are included into the sormas release ZIP file. Then it is ensured the actual version is running into the keycloak @JonasCir

**Comment by JonasCir** [comment]
Good point thanks. Since these files don't exist after Keycloak 17 anyways I'll leave it with that for now as they won't change in the meantime. Once https://github.com/hzi-braunschweig/SORMAS-Project/issues/8018 lands, I will include all the necessary files in the release.
